jekyll/_cci2/use-the-cli-for-config-policy-management.adoc

```diff
@@ -0,0 +1,226 @@
+
+image:org-id.png[screenshot showing where to find your organization ID]
+====
+
+== Create and upload a policy 
+
+Follow these steps to create a policy that checks the `version` of CircleCI config files to ensures it is greater than or equal to `2.1`.
+
+. Create a policy file in an empty directory. For example:
++
+[source,shell]
+----
+mkdir ./config-policies
+----
+
+. Inside your new directory create a `rego` file for our new policy, for example, `version.rego`.
+
+. Add the following content to `version.rego`:
++
+[source,rego]
+----
+# All policies start with the org package definition
+package org
+
+policy_name["example"]
+
+# signal to circleci that check_version is enabled and must be included when making a decision
+enable_rule["check_version"]
+
+# signal to circleci that check_version is a hard_failure condition and that builds should be
+# stopped if this rule is not satisfied.
+hard_fail["check_version"]
+
+# define check version
+check_version = reason {
+    not input.version # check the case where version is not in the input
+    reason := "version must be defined"
+} {
+    not is_number(input.version) # check that version is number
+    reason := "version must be a number"
+} {
+    not input.version >= 2.1 # check that version is at least 2.1
+    reason := sprintf("version must be at least 2.1 but got %s", [input.version])
+}
+----
+
+. Upload your policy to your organization:
++
+[source,shell]
+----
+circleci-cli policy push ./config --owner-id <your-organization-ID>
+----
++
+Now, when a pipeline is triggered within your organization, the project's `.circleci/config.yml` will be validated against this policy.
+
+== Update a policy
+
+To illustrate making a change to an existing policy, suppose you made an error when creating that policy, and that configs in your organization are using CircleCI config version `2.0` and that you want your policy to reflect this.
+
+. Change the rule definition in your `version.rego` file:
++
+[source,rego]
+----
+{
+    not input.version >= 2.0 # check that version is at least 2.0
+    reason := sprintf("version must be at least 2.0 but got %s", [input.version])
+}
+----
+
+. Push the policy directory containing the updated policy file using the CLI (verify the diff, and choose yes when prompted):
++
+[source,shell]
+----
+circleci-cli policy push ./config --owner-id <your-organization-ID>
+----
+
+== Manage policies with your VCS
+
+CircleCI policies are managed by pushing directories of policies to CircleCI via the CLI. We recommend managing your policy directory via your VCS by storing your policies in a repository within your organization.
+
+This is how policies are managed internally at CircleCI. Pushing policy bundles is done by triggering CircleCI pipelines.
+
+We also recommend creating a bot account for pushing policies and to use its associated CircleCI token. The context should be restricted to groups that are responsible for managing policies. See Restricted Contexts.
```

Comment:
Is a "bot account" a machine user? Is this recommendation because people come and go so you don't want this tied to a specific person?

Reply:
@davidmparrott

Reply:
That and also so that you don't have credential leaking. The idea is that most devs will likely not have access to make changes to this repo, or to only make changes to subsections of it that might be specifically relevant to their team. By having a bot (machine) user you're able to limit the scope of who can and can't manage the policies for your organization

Reply:
That, and that, and also it would also help distinguish in the audit logs that a particular policy was created by bot account which can indicate that it was created via `merge to main` in policy VCS (in contrast to being push manually using CLI, which would almost always be done by a human user's token).

Reply:
I like the term "bot account" more, but apparently GitHub (and therefore CCI) refers it to as machine user.
https://circleci.com/docs/github-integration#controlling-access-via-a-machine-user

As for decoupling, I would like "bot account".
@davidmparrott ?